src/lib/createPollBlocks.ts

```diff
@@ -35,15 +35,15 @@ export function createPollBlocks(block: BlockBuilder, question: string, options:
     if (poll.activeLivePoll && !poll.finished) {
         block.addContextBlock({
             elements: [
-                block.newMarkdownTextObject(`The poll will finish at ${poll.livePollEndTime}`),
+                block.newMarkdownTextObject(`The poll will finish at *${poll.livePollEndTime}*`),
```

Comment:
Here's an idea: Instead of showing the UTC-0 time, why not we use the poll creator's timezone here? 

Tip: You can get any user's timezone offset from `IUser.utcoffset` prop in apps-engine ;)

Reply:
Yeah, that is a great idea, I'll make that changes.

Reply:
I try to come up with the solution please have a look!